server/app/views/errors/ErrorComponent.java

```diff
@@ -70,6 +70,6 @@ public static DivTag renderErrorComponent(
                                                             MessageKey.ERROR_STATUS_CODE
                                                                 .getKeyName(),
                                                             statusCode.orElse("")))
-                                                        .withClass("text-base")))))));
+                                                        .withClass("text-base-dark")))))));
```

Comment:
This fixes a contrast issue on the error page now that uswds is generating color classes.